clients/java/client/src/main/java/org/camunda/bpm/client/UrlResolver.java

```diff
@@ -0,0 +1,9 @@
+package org.camunda.bpm.client;
```

Comment:
This file is missing the licenseHeader, can you please add it to the top of this file. You can copy it from one of the other files